app/controllers/sessions_controller.rb

```diff
@@ -2,13 +2,19 @@ class SessionsController < ApplicationController
   # If you're using a strategy that POSTs during callback, you'll need to skip the authenticity token check for the callback action only. 
   # skip_before_action :verify_authenticity_token, only: :create
 
+  # TODO: delete a user
   def new
     # This route is catched by the Omniauth Middleware and is invisible to rake routes
+    # TODO: check if user is already in database
     redirect_to '/auth/github'
   end
 
   def create
-    User.find_or_create_from_auth_hash(auth_hash)
+    user = User.find_or_create_from_auth_hash(auth_hash)
+    str = [user.id.to_s, user.name, user.email].to_s
```

Comment:
do you need the to_s  on user.id as well as on the array?

Reply:
yes to make the id which is an integer a string when parsed and to make the array an string